src/connection.rs

```diff
@@ -248,6 +253,15 @@ pub fn connect(peer_contact: StaticContactInfo,
     Err(last_err)
 }
 
+/*  FIXME(canndrew) 23/03/2016
```

Comment:
you can put a `#[allow(unused)]`/`#[warn(unused)]` just before the function that should ignore the lint instead commenting. But I'm not sure about the trade-off (compilation time of unused function or maintainability of not breaking commented code accidentally). As it'll be enabled very soon, I don't mind about keeping the code commented.